src/main/java/org/fundacionjala/pivotal/api/APIProjectsRequests.java

```diff
@@ -0,0 +1,75 @@
+package org.fundacionjala.pivotal.api;
+
+import com.jayway.restassured.response.Response;
+import java.util.Map;
+import static com.jayway.restassured.RestAssured.given;
+
+/**
+ * APIProjectsRequests.java
+ * Class that implements the methods to get, post, delete and put through Pivotal API.
+ */
+public final class APIProjectsRequests {
+    /**
+     * Constructor.
+     */
+    private APIProjectsRequests() {
+    }
+
+    /**
+     * static attribute to manage project's id through processes.
+     */
+    private static String projectId = "0";
+
+    /**
+     * Response attribute to get request.
+     */
+    private static Response response;
+
+    /**
+     * Method to post a request.
+     * @param values map.
+     * @param endpoint to request to.
+     */
+    public static void postNewProject(final Map<String, String> values, final String endpoint) {
+        response = given()
+                .spec(APICommons.buildRequestHeader())
+                .params(values)
+                .when()
+                .post(endpoint);
+    }
+
+    /**
+     * Method to delete a request.
+     * @param endpoint to request to.
+     */
+    public static void deleteProject(final String endpoint) {
+        //System.out.println("projectId:" + projectId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)